tests/common.py

```diff
@@ -14,7 +14,7 @@
 
 TEST_CONN = os.getenv('SQLA_TEST_CONN', 'postgresql:///cfdm_unit_test')
 
-rest.app.config['NPLUSONE_RAISE'] = True
+rest.app.config['NPLUSONE_RAISE'] = False
```

Comment:
That's what I thought - I'll have to investigate and see what's going on locally for me.